rebar.config

```diff
@@ -32,10 +34,6 @@
 	     [{relx, [{dev_mode, false},
 		      {include_erts, true},
 		      {include_src, false}
-		     ]},
```

Comment:
Would we want to replicate these deps in the test profile, instead of adding them in by default?

Reply:
"test/experimental" builds are the ones not marked with "prod". its not meant to have them in the "test" build for which the unit tests are running.

its really for the build-artifact that one runs if one works on a capwap setup.